build.dependencies.gradle

```diff
@@ -45,8 +44,8 @@ dependencies {
     implementation 'com.acmerobotics.dashboard:dashboard:0.4.16'
 
     // Adding support for Pedro Pathing
-    implementation 'com.pedropathing:ftc:2.0.1'
-    implementation 'com.pedropathing:telemetry:0.0.6'
+    implementation 'com.pedropathing:ftc:2.0.2'
```

Comment:
I need to make these same changes to 16750. Having different dependency versions in the same repo is never a good thing if it can be avoided...

Reply:
lol. Didn't realize this was already in the shared .gradle file 😆